command/render.go

```diff
@@ -0,0 +1,99 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/jrasell/levant/levant"
+	"github.com/jrasell/levant/logging"
+)
+
+// RenderCommand is the command implementation that allows users to render a
+// Nomad job template based on passed templates and variables.
+type RenderCommand struct {
+	args []string
+	Meta
+}
+
+// Help provides the help information for the template command.
+func (c *RenderCommand) Help() string {
+	helpText := `
+Usage: levant render [options] TEMPLATE
+
+  Render a Nomad job template, useful for debugging.
+
+General Options:
+
+  -log-level=<level>
+    Specify the verbosity level of Levant's logs. Valid values include DEBUG,
+    INFO, and WARN, in decreasing order of verbosity. The default is INFO.
+
+  -var-file=<file>
+	Specify the variable files used in rendering the template.
+	
+  -out=<file>
+	Specify the path to write the rendered template out to, set to "-" to use stdout,
+	if an file exists at the specified path it will be truncated before rendering.
+	The default is "-".
+`
+	return strings.TrimSpace(helpText)
+}
+
+// Synopsis is provides a brief summary of the template command.
+func (c *RenderCommand) Synopsis() string {
+	return "Render a Nomad job from a template"
+}
+
+// Run triggers a run of the Levant template functions.
+func (c *RenderCommand) Run(args []string) int {
+
+	var variables, addr, log, outPath string
+	var err error
+	var tpl *bytes.Buffer
+
+	flags := c.Meta.FlagSet("build", FlagSetVars)
```

Comment:
"render" rather than "build". I will fix the error within the deploy Meta.FlagSet